_posts/2018-04-10-building-the-peer-to-peer-internet.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Building The Peer-to-Peer Internet
+slug: building-the-peer-to-peer-internet
+text: Workshop series at the Toronto Public Library
+location: Fort York branch of the Toronto Public Library
+locationLink: https://www.torontopubliclibrary.ca/detail.jsp?Nr=p_cat_branch_name:Fort%20York
+date: 2018-04-10
+startTime: '18:30'
+endTime: '20:00'
+---
+
+Toronto Mesh will be presenting the workshop series **{{ page.title }}** each Tuesday from April 10 to May 15, 2018 at the [{{ page.location }}]({{ page.locationLink }}).
+
+Participants will learn about community networks, wireless mesh networks, distributed applications, and train their pet Raspberry Pi to do networking things. While this course assumes no technical expertise, all sessions contain a technical hands-on component and a reflection activity to explore the role of technologies on our society. A socially-minded tinkerer who enjoys a collaborative learning experience will be the ideal participant.
+
+The course will cover the following topics:
+
+* Tue Apr 10 - *What is mesh?*
+* Tue Apr 17 - *Setting up your first node*
+* Tue Apr 24 - *Building nodes (hardware)*
+* Tue May 01 - *Planning a network*
+* Tue May 08 - *Measuring networks*
+* Tue May 15 - *Living on a decentralized network*
+
+Full syllabus is [available here](https://github.com/tomeshnet/tpl-workshop).
```

Comment:
I would just link the "full syllabus" rather than "here" (thinking about these links being meaningful if stripped of context)